docs/cri/registry.md

```diff
@@ -185,3 +181,8 @@ DEBU[0000] PullImageRequest: &PullImageRequest{Image:&ImageSpec{Image:gcr.io/you
 DEBU[0001] PullImageResponse: &PullImageResponse{ImageRef:sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42,}
 Image is up to date for sha256:78096d0a54788961ca68393e5f8038704b97d8af374249dc5c8faec1b8045e42
 ```
+
+---
+
+NOTE: The configuration syntax used in this doc is in version 2 which is the recommended since `containerd` 1.3. For the previous config format you can reference [https://github.com/containerd/containerd/blob/v1.4.4/PLUGINS.md#configuration-1](https://github.com/containerd/containerd/blob/v1.4.4/PLUGINS.md#configuration-1).
```

Comment:
believe it or not for this registry doc.. the 1.2 version was in continerd/cri .. over here:
https://github.com/containerd/cri/blob/release/1.2/docs/registry.md